buku.py

```diff
@@ -67,6 +67,16 @@
 logdbg = logger.debug
 logerr = logger.error
 
+def read_input(msg):
```

Comment:
Please make the name shorter, like `read_in()`. Some lines are crossing 80 chars.